x/gov/proposals.go

```diff
@@ -351,16 +346,16 @@ func (status ProposalStatus) Format(s fmt.State, verb rune) {
 	}
 }
 
-//-----------------------------------------------------------
 // Tally Results
+// TODO: Can these be sdk.Int?
```

Comment:
If you fixed this, remove the TODO?

Reply:
done

Reply:
circleci build is progressing very slowly - please let's resist the urge to add commits bringing cosmetic fixes only for now, we can address these kinds of things in another PR into develop